easybuild/easyconfigs/g/gompi/gompi-2015b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2015b'
+
+homepage = '(none)'
```

Comment:
Hi folks,

if you recall, I offered that page back then when working in the `readthedocs` act,
exactly to make it usable in cases like this. It's the kind of page you need when
you first encounter gompi and you ask yourself: *what the heck is toolchain gompi?*!